WordPress/Classes/Utility/AuthTokenIssueSolver.swift

```diff
@@ -47,8 +47,16 @@ struct AuthTokenIssueSolver {
     }
 
     private func showCancelReAuthenticationAlert(in window: UIWindow, onDeletionConfirmed: @escaping () -> Void) {
-        let title = NSLocalizedString("Careful!", comment: "Title for the warning shown to the user when he refuses to re-login when the authToken is missing.")
-        let message = NSLocalizedString("Proceeding will remove all WordPress.com data from this device, and delete any locally saved drafts. You will not lose anything already saved to your WordPress.com blog(s).", comment: "Message for the warning shown to the user when he refuses to re-login when the authToken is missing.")
+        let title = NSLocalizedString(
+            "auth_token_issue_solver.cancel_alert.title",
```

Comment:
Hey, thanks for updating it! Ideally, I would suggest moving these under `private enum Strings` - this is what we typically do. And it [should used](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys) camel case.